src/index.js

```diff
@@ -1,11 +1,6 @@
-import app from './app';
+import React from 'react';
+import { render } from 'react-dom';
+// eslint-disable-next-line import/extensions
```

Comment:
I think we can set the eslint rule in the config file to do this.

Reply:
Yes @Simplemart17 we can do that on the next PR or what do you think?